.gitignore

```diff
@@ -5,4 +5,5 @@ npm-debug.log
 .env
 coverage
 lib
-.DS_Store
\ No newline at end of file
+.DS_Store
+dist
```

Comment:
The code must be compiled with `@vercel/ncc` and hosted on the repo for the GitHub Action to run it.

Reply:
fixed in latest commits.